packages/turf-meta/index.d.ts

```diff
@@ -203,7 +206,7 @@ export function segmentReduce<Reducer extends any, P = Properties>(
     | Feature<GeometryCollection, P>
     | GeometryCollection,
   callback: (
-    previousValue?: Reducer,
+    previousValue: Reducer,
```

Comment:
According to docs, this will always have a value, either it will be the initial value provided by the caller, or it will be the first element of the array